### PR TITLE
Add content restricted meta actions

### PR DESCRIPTION
**Pull request by tannerm**
#880

### CODE REVIEW

File: includes/admin/metabox-view.php [hunk 1]
```diff
@@ -70,6 +70,7 @@
 		</select>
 	</p>
 </div>
+<?php do_action( 'rcp_metabox_field_options_before' ); ?>
 <div id="rcp-metabox-field-options" class="rcp-metabox-field">
```
Comment (mindctrl) [inline includes/admin/metabox-view.php]: @tannerm can you base this off release/2.7? We added a filter in metabox-view.php, and that would help with any possible merge conflicts.

File: includes/admin/metabox.php [hunk 2]
```diff
@@ -192,5 +192,7 @@ function rcp_save_meta_data( $post_id ) {
 	}
 	update_post_meta( $post_id, '_is_paid', $is_paid );
 
+	do_action( 'rcp_save_meta_data' );
```
Comment (mindctrl) [inline includes/admin/metabox.php]: @tannerm probably good to pass the post_id to this?

Comment (tannerm) [inline includes/admin/metabox.php]: @mindctrl oh! Of course. One sec.

File: includes/admin/metabox-view.php (hunk 1 above)
Comment (tannerm) [inline includes/admin/metabox-view.php]: Done

File: includes/admin/metabox.php (hunk 2 above)
Comment (tannerm) [inline includes/admin/metabox.php]: Just updated

**Comment by tannerm** [comment]
Found a merge conflict with 2.7 and just merged it in to this PR.

File: includes/admin/metabox-view.php [hunk 3]
```diff
@@ -70,6 +70,8 @@
 		</select>
 	</p>
 </div>
+<?php do_action( 'rcp_metabox_field_options_before' ); ?>
+<div id="rcp-metabox-field-options" class="rcp-metabox-field">
```
Comment (mindctrl) [inline includes/admin/metabox-view.php]: @tannerm just noticed the duplicated div here

**Pull request closed without merging**